screens/user/user-list.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import {useSelector} from 'react-redux';
+import {Wrapper, Text, View} from './styled';
+
+export const UserListComponent = () => {
+  const {users} = useSelector((state) => state.user);
```

Comment:
Please define another function as selector that does the picking of state node from it's tree.